### ISSUE TITLE
Select labeling issues

### ISSUE THREAD

**Issue by crisbeto** [angular/material2#6382]
Based on the select accessibility assessment:
* In NVDA the select value isn't announced when focusing.
* The collapsed/open state isn't announced.
* The select group items are not announced, in addition to the options not being grouped together.

**Comment by jelbourn** [comment]
Tracking via #11083
